client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/submissionFormValidate.js

```diff
@@ -0,0 +1,47 @@
+import { defineMessages } from 'react-intl';
+import { questionTypes } from '../../constants';
+
+const translations = defineMessages({
+  pleaseRecordYourVoice: {
+    id: 'course.assessment.submission.pages.SubmissionEditIndex.submissionFormValidate.pleaseRecordYourVoice',
+    defaultMessage: 'Please record your voice',
+  },
+});
+
+const validateDefaultQuestion = () => undefined;
+
+const validateVoiceResponse = (values) => {
+  const { file: fileValue } = values;
+  const { file, url } = fileValue;
+  if (url || Object.prototype.toString.call(file) === '[object File]') {
```

Comment:
Any better ways to check it is a file?

Reply:
@huangjosh any updates on this?

Reply:
@allenwq I think ```instanceof``` would be a better choice. Fixed, and updated. the PR.